sematic/calculator.py

```diff
@@ -254,9 +254,40 @@ def func(
     resource_requirements: Optional[ResourceRequirements] = None,
     retry: Optional[RetrySettings] = None,
     base_image_tag: Optional[str] = None,
-) -> Union[Callable, Calculator]:
+) -> Union[Calculator, Callable]:
     """
-    Sematic Function decorator.
+    The Sematic Function decorator.
+
+    This is used to instruct that the instrumented Python function is a
+    top-level step in the pipeline. The function's execution will be tracked
```

Comment:
Remove "top-level"?

Reply:
FWIW I typically describe it as "this identifies the code as a unit of work that Sematic knows about for tracking and scheduling"

Reply:
Used suggested formulation.